docs/modules/gis/api-reference/geojson-to-binary.md

```diff
@@ -121,9 +122,10 @@ corresponds to 3D coordinates, where each vertex is defined by three numbers.
 
 | Option           | Type      | Default           | Description                                                                                                                                             |
 | ---------------- | --------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| fixRingWinding   | `Boolean` | `true`            | Whether the fix incorrect ring winding for polygons. Valid `GeoJSON` polygons have the outer ring coordinates in CCW order and with holes in CW order   |
+| fixRingWinding   | `Boolean` | `true`            | Whether to fix incorrect ring winding for polygons. Valid `GeoJSON` polygons have the outer ring coordinates in CCW order and with holes in CW order    |
 | numericPropKeys  | `Array`   | Derived from data | Names of feature properties that should be converted to numeric `TypedArray`s. Passing `[]` will force all properties to be returned as normal objects. |
 | PositionDataType | `class`   | `Float32Array`    | Data type used for positions arrays.                                                                                                                    |
+| triangulate      | `Boolean` | `true`            | Whether polygons are triangulated as part of the conversion                                                                                             |
```

Comment:
```suggestion
| triangulate      | `Boolean` | `true`            | Whether polygons are tessellated (broken into triangles) as part of the conversion. |
```

Reply:
Add to the notes below:

- Tessellation of polygons can be time consuming. If not needed, set the `tessellate` option to false.

Reply:
Will add note. Regarding the naming, my view is that we should stick to `triangulate` because:

- Consistency with data format, `triangulate` produces the `triangles` array
- Earcut specifically describes itself as a "polygon triangulation library"
- The output is a list of triangles, whereas tesselation is a more general term, i.e. one can tesselate an area with squares/hexagons